docs/index.md

```diff
@@ -68,7 +68,49 @@ Optional settings:
 }
 ```
 #### Sample configuration
-The sample configuration should be a YML file which adheres to the following
+
+##### Verification
+
+All samplesheet formats can be verified using `biowdl-input-converter`. 
+It can be installed with `pip install biowdl-input-converter` or 
+`conda install biowdl-input-converter` (from the bioconda channel). 
+Python 3.7 or higher is required.
+
+With `biowdl-input-converter --validate samplesheet.csv` The file
+"samplesheet.csv" will be checked. Also the presence of all files in
+the samplesheet will be checked to ensure no typos were made. For more
+information check out the [biowdl-input-converter readthedocs page](
+https://biowdl-input-converter.readthedocs.io).
+
+##### CSV Format
+The sample configuration can be given as a csv file with the following 
+columns: sample, library, readgroup, R1, R1_md5, R2, R2_md5.
+
+column name | function
+---|---
+sample | sample ID
+library | library ID. These are the libraries that are sequenced. Usually there is only one library per sample
+readgroup | readgroup ID. Usually a library is sequenced on multiple lanes in the sequencer, which gives multiple fastq files (referred to as readgroups). Each readgroup pair should have an ID.
+R1| The fastq file containing the forward reads 
```

Comment:
Calling these `forward reads` might actually be confusing, technically it depends on the protocol whether these are the forward or the reverse reads (or some mixture), hence why you need to specify the strandedness in the inputs.